test/it/crdvalidation/etcd/specupdate_test.go

```diff
@@ -0,0 +1,216 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package etcd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gardener/etcd-druid/test/utils"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	. "github.com/onsi/gomega"
+)
+
+// tests for the Update Validations:
+
+// etcd.spec.storageClass is immutable
+func TestValidateSpecStorageClass(t *testing.T) {
```

Comment:
```suggestion
func TestValidateUpdateSpecStorageClass(t *testing.T) {
```

Use `TestValidateUpdate` prefix for all update tests, to make it easier to run only update validation tests via `go test` command.